client/src/location_data.js

```diff
@@ -0,0 +1,67 @@
+  { label: 'Afula', value: 'Afula' },
+  { label: 'Arad', value: 'Arad' },
+  { label: 'Ariel', value: 'Ariel' },
+  { label: 'Ashdod', value: 'Ashdod' },
+  { label: 'Ashkelon', value: 'Ashkelon' },
+  { label: 'Bat Yam', value: 'Bat Yam' },
+  { label: 'Beersheba', value: 'Beersheba' },
+  { label: 'Beit Shean', value: 'Beit Shean' },
+  { label: 'Beit Shemesh', value: 'Beit Shemesh' },
+  { label: 'Beitar Illit', value: 'Beitar Illit' },
+  { label: 'Bnei Brak', value: 'Bnei Brak' },
+  { label: 'Dimona', value: 'Dimona' },
+  { label: 'Eilat', value: 'Eilat' },
+  { label: 'Elad', value: 'Elad' },
+  { label: 'Givat Shmuel', value: 'Givat Shmuel' },
+  { label: 'Givatayim', value: 'Givatayim' },
+  { label: 'Hadera', value: 'Hadera' },
+  { label: 'Haifa', value: 'Haifa' },
+  { label: 'Herzliya', value: 'Herzliya' },
+  { label: 'Hod HaSharon', value: 'Hod HaSharon' },
+  { label: 'Holon', value: 'Holon' },
+  { label: 'Jerusalem', value: 'Jerusalem' },
+  { label: 'Karmiel', value: 'Karmiel' },
+  { label: 'Kiryat Ata', value: 'Kiryat Ata' },
+  { label: 'Kiryat Bialik', value: 'Kiryat Bialik' },
+  { label: 'Kiryat Gat', value: 'Kiryat Gat' },
+  { label: 'Kiryat Malakhi', value: 'Kiryat Malakhi' },
+  { label: 'Kiryat Motzkin', value: 'Kiryat Motzkin' },
+  { label: 'Kiryat Ono', value: 'Kiryat Ono' },
+  { label: 'Kiryat Shmona', value: 'Kiryat Shmona' },
+  { label: 'Kiryat Yam', value: 'Kiryat Yam' },
+  { label: 'Kfar Saba', value: 'Kfar Saba' },
+  { label: 'Lod', value: 'Lod' },
+  { label: 'Maale Adumim', value: 'Maale Adumim' },
+  { label: 'Maalot-Tarshiha', value: 'Maalot-Tarshiha' },
+  { label: 'Migdal HaEmek', value: 'Migdal HaEmek' },
+  { label: 'Modiin-Maccabim-Reut', value: 'Modiin-Maccabim-Reut' },
+  { label: 'Modiin Illit', value: 'Modiin Illit' },
+  { label: 'Nahariya', value: 'Nahariya' },
+  { label: 'Nazareth Illit', value: 'Nazareth Illit' },
+  { label: 'Nesher', value: 'Nesher' },
+  { label: 'Ness Ziona', value: 'Ness Ziona' },
+  { label: 'Netanya', value: 'Netanya' },
+  { label: 'Netivot', value: 'Netivot' },
+  { label: 'Ofakim', value: 'Ofakim' },
+  { label: 'Or Akiva', value: 'Or Akiva' },
+  { label: 'Or Yehuda', value: 'Or Yehuda' },
+  { label: 'Petah Tikva', value: 'Petah Tikva' },
+  { label: 'Raanana', value: 'Raanana' },
+  { label: 'Ramat Gan', value: 'Ramat Gan' },
+  { label: 'Ramat HaSharon', value: 'Ramat HaSharon' },
+  { label: 'Ramla', value: 'Ramla' },
+  { label: 'Rehovot', value: 'Rehovot' },
+  { label: 'Rishon LeZion', value: 'Rishon LeZion' },
+  { label: 'Rosh HaAyin', value: 'Rosh HaAyin' },
+  { label: 'Safed', value: 'Safed' },
+  { label: 'Sderot', value: 'Sderot' },
+  { label: 'Tel Aviv-Yaffo', value: 'Tel Aviv-Yaffo' },
+  { label: 'Tiberias', value: 'Tiberias' },
+  { label: 'Tirat Carmel', value: 'Tirat Carmel' },
+  { label: 'Yavne', value: 'Yavne' },
+  { label: 'Yehud-Monosson', value: 'Yehud-Monosson' },
+  { label: 'Yokneam', value: 'Yokneam' },
```

Comment:
@hoslack It's great that you've found all possible locations and imported them but please remember that we said we would focus on 3 cities for the MVP. The chosen cities are the ones we discussed with Amira, the product owner.

Reply:
There's no harm in having the data there. It took me ages and effort to prepare the data. Please allow me to keep ti there!

Reply:
Holy hell that's a big amount of data it's just going to look awful when selecting you sure about this now @hoslack ?

( you probably going to say yes cause you can't be bothered if changing i've been there mate )